stream-parent/stream-client/src/main/java/org/interledger/stream/sender/DefaultBackoffController.java

```diff
@@ -0,0 +1,109 @@
+package org.interledger.stream.sender;
+
+import static org.interledger.core.InterledgerErrorCode.T04_INSUFFICIENT_LIQUIDITY;
+
+import org.interledger.core.InterledgerErrorCode;
+import org.interledger.core.InterledgerPreparePacket;
+import org.interledger.core.InterledgerRejectPacket;
+import org.interledger.core.InterledgerResponsePacket;
+import org.interledger.link.Link;
+import org.interledger.link.exceptions.LinkRetriesExceededException;
+
+import io.github.resilience4j.retry.IntervalFunction;
+import io.github.resilience4j.retry.Retry;
+import io.github.resilience4j.retry.RetryConfig;
+
+import java.time.Duration;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Function;
+
+public class DefaultBackoffController implements BackoffController {
```

Comment:
If we keep this PR, let's add some Javadoc to outline what the default backoff behavior is?